librad/src/git/identities/project.rs

```diff
@@ -143,6 +151,12 @@ pub fn merge(storage: &Storage, urn: &Urn, from: PeerId) -> Result<Project, Erro
     Ok(next)
 }
 
+pub fn is_fork(storage: &Storage, left: &Urn, right: &Urn) -> Result<Fork, Error> {
+    let left = get(storage, left)?.ok_or_else(|| Error::NotFound(left.clone()))?;
+    let right = get(storage, right)?.ok_or_else(|| Error::NotFound(right.clone()))?;
+    Ok(identities(&storage).is_fork(left.revision.into(), right.revision.into())?)
```

Comment:
This passes two `tree` oids, but `Identities::is_in_ancestry_path` expects the first to be a `commit`, and the second to be a `tree` oid.

Reply:
Ah, confusion on my part :+1:  Is `content_id` the commit oid?